src/fragmentarium/infrastructure/FragmentRepository.ts

```diff
@@ -54,10 +86,14 @@ function createFragment(dto): Fragment {
     references: dto.references.map(createReference),
     uncuratedReferences: dto.uncuratedReferences,
     genres: Genres.fromJson(dto.genres),
-    introduction: dto.introduction,
```

Comment:
Since we don't need any processing for the `introduction`, it's not needed (already included in `...dto`)